plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -353,6 +355,8 @@ private TypeAndDecoder toTrino(String prefix, IndexMetadata.Field field)
                 case "text":
                 case "keyword":
                     return new TypeAndDecoder(VARCHAR, new VarcharDecoder.Descriptor(path));
+                case "geo_point":
+                    return new TypeAndDecoder(VARCHAR, new GeopointDecoder.Descriptor(path));
```

Comment:
I don't think we want to map geo_point to varchar type. It would be better to map to `SphericalGeography` type or `row(longitude, latitude)` or add a new type in Elasticsearh module likes MongoDB's ObjectId type.